packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx

```diff
@@ -103,6 +104,12 @@ function ViewDetailed(props: ViewDetailedProps) {
     const stableTokens = useStableTokensDebank()
     const chainDetailed = useChainDetailed()
 
+    const etherPrice = useValueRef(currentEtherPriceSettings)
+
+    console.log({
```

Comment:
![image](https://user-images.githubusercontent.com/52657989/126523872-ca873f00-a614-4d7d-81d9-8afd3f9f6ef3.png)

CC @zhouhanseng Can we use it to replace the previous plan?